src/ros_commo.py

```diff
@@ -308,30 +308,19 @@ def chat_perceived_text_cb(self, chat_heard):
 	# started, or stopped vocalizing.  This message might be published
 	# by either the TTS module itself, or by some external chatbot.
 	#
-	# Unit test by saying
-	#    rostopic pub --once chat_events std_msgs/String speechstart
-	#    rostopic pub --once chat_events std_msgs/String speechend
```

Comment:
Can you restore these comments? They are useful for debugging, to manually verify that the system is working correctly.